tsconfig.json

```diff
@@ -10,7 +10,14 @@
     "jsx": "react",
     "strict": true,
     "noEmit": true,
-    "outDir": "./out"
+    "outDir": "./out",
+    "lib": [
```

Comment:
since this is now specified explicitly, i've included what was the previous default, plus `esnext.asynciterable`

> Note: If --lib is not specified a default list of libraries are injected. The default libraries injected are:
► For --target ES5: DOM,ES5,ScriptHost
► For --target ES6: DOM,ES6,DOM.Iterable,ScriptHost

from https://www.typescriptlang.org/docs/handbook/compiler-options.html